src/View/UIViewTrait.php

```diff
@@ -3,20 +3,32 @@
 namespace BootstrapUI\View;
 
 /**
- * UIViewTrait: Trait that loads the custom UIBootstrap helpers and sets View
- * layout to the UIBootstrap's one.
+ * ## UIViewTrait:
+ *
+ * - loading the custom UIBootstrap helpers.
+ * - setting the View layout to the UIBootstrap's one.
  */
 trait UIViewTrait
 {
     /**
      * Initialization hook method.
      *
-     * @return void
+     * ### Options: an array of keys, available keys are:
+     * - `layout`:
+     *      - If empty or true will use the plugin's layout
+     *      - If a layout name passed it will be used
+     *      - If false do nothing ( will keep your layout )
+     *
+     * @return  void
      */
-    public function initializeUI()
+    public function initializeUI(array $options = [])
     {
-        if (empty($this->layout) || $this->layout === 'default') {
+        if (!array_key_exists('layout', $options)) {
```

Comment:
You are really testing my patience. I gave the exact code to use and yet you did something different.